sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ReloadWorkflowTasks.scala

```diff
@@ -85,7 +85,7 @@ object ReloadWorkflowTasks {
   def writeLauncherTask(stage: TaskKey[Attributed[File]]): Def.Initialize[Task[File]] =
     Def.task {
       val entryPoint =
-        (mainClass in (scalaJSLauncher in stage)).value.getOrElse(sys.error("No main class detected"))
+        (mainClass in (scalaJSMainModuleInitializer in stage)).value.getOrElse(sys.error("No main class detected"))
```

Comment:
This doesn't make any sense. This is equivalent to `mainClass in scalaJSMainModuleInitializer`. And that is also probably too complex and should be just `mainClass`, because the value of `mainClass` is only relevant in the configuration axis; the task axis is not used for this key either by sbt or Scala.js' sbt plugin.

Note that this comment applies both to before and after your changes.